trigger-mender-conductor-enterprise.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -u -x -e
+
+body='{
+"request": {
+"branch":'"$TRAVIS_BRANCH"'
```

Comment:
add env with mender-conductor revision (eg `COMMIT` variable content)
Outside is branch / tag builds, master in mender-conductor-enterprise needs to produce: master_<mender-conductor-sha>_ <mender-contuctor-enterprise-sha> to have unique tags for master for the combination of those two

Reply:
You can set arbitrary envs accrdign to docs:

```
body='{
 "request": {
 "message": "Override the commit message: this is an api request",
 "branch":"master",
 "config": {
   "env": {
     "matrix": ["TEST=unit"]
   },
   "script": "echo FOO"
  }
}}'
```

Reply:
https://docs.travis-ci.com/user/triggering-builds/